assets/js/blocks/classic-template/types.ts

```diff
@@ -1 +1,19 @@
+/**
+ * External dependencies
+ */
+import { type BlockInstance } from '@wordpress/blocks';
+
 export type TemplateDetails = Record< string, Record< string, string > >;
+
+export type InheritedAttributes = {
+	align?: string;
+};
+
+export type BlockifiedTemplate = {
```

Comment:
```suggestion
export type BlockifiedTemplateConfig = {
```

I think a more explicit name here will help other folks understand this easier.